packages/ipfs/src/index.js

```diff
@@ -113,7 +113,7 @@ async function post(gateway, json, rawHash) {
 //   return decrypted.data
 // }
 
-async function getTextFn(gateway, hashAsBytes) {
+async function getTextFn(gateway, hashAsBytes, timeoutMS) {
```

Comment:
should we set the default for this too?

Reply:
There aren't any direct consumers of this function according to my grepping. Just internal in `@origin/ipfs`.